app/services/push_notification_service.rb

```diff
@@ -0,0 +1,32 @@
+class PushNotificationService
+  @@server_key = 'xxx'
```

Comment:
Need to get from firebase console
firebase console > project setting > cloud messaging > server key